models/warehouses_type2_dimension.sql

```diff
@@ -36,6 +46,11 @@ order by timestamp) as _valid_to
 )
 
 select
+    {% if var('uses_org_view', false) %}
+    warehouse_snapshots.organization_name,
+    warehouse_snapshots.account_name,
+    warehouse_snapshots.account_locator,
+    {% endif %}
```

Comment:
Would you be able to simplify this significantly if you set this to always include these new columns? I'm not sure the conditional rendering is actually necessary. In the single account version, they'd just be for one account, but it still works